src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/changelist/additions/LegalPersonBeneficialOwnerAddition.java

```diff
@@ -30,6 +38,12 @@ public void setCorporateName(String corporateName) {
         this.corporateName = corporateName;
     }
 
+    public Address getRegisteredOffice() {return this.registeredOffice; }
```

Comment:
I think this isn't allowed under [4.1.2 Nonempty blocks: K & R style](https://google.github.io/styleguide/javaguide.html#s4.1.2-blocks-k-r-style) in the Google Java Style Guide which CompaniesHouse uses.